plutus-book/doc/authors.adoc

```diff
@@ -1,4 +1,5 @@
 [#authors]
+:sectnums!:
```

Comment:
Better to do this in the spine, I think, so it's clear where it goes. I'll look at that.

Reply:
:+1: